tap-snapshots/test-snyk-to-html.test.ts-TAP.test.js

```diff
@@ -137,8 +138,17 @@ exports[`test/snyk-to-html.test.ts TAP template output displays vulns in descend
       padding-bottom: 1px;
     }
   
+    .paths {
+      margin-left: 8px;
+    }
+    .header-wrap {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      padding-top: 2em;
+    }
     .project__header {
-      background-color: #4C4A73;
+      background-color: #4b45a9;
```

Comment:
`background-image: linear-gradient(119.42deg, rgb(112, 56, 159), rgb(49, 53, 1` ?

Reply:
Why image? The colour was taken from logged in snyk app

Reply:
it's noticeable when you follow link to snyk.io (not app.snyk.io) - which uses a gradient